src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxFactoryTests.cs

```diff
@@ -27,6 +27,18 @@ public void AlternateVerbatimString()
             Assert.Equal("$@\"", token.ValueText);
         }
 
+        [Fact]
+        public void UsingDirective()
+        {
+            string name = "System.String";
+            var someValidName = SyntaxFactory.ParseName(name);
+            var usingDirective = SyntaxFactory.UsingDirective(SyntaxFactory.Token(SyntaxKind.StaticKeyword), null, someValidName);
+            Assert.NotNull(usingDirective);
+            Assert.Equal(SyntaxKind.StaticKeyword, usingDirective.StaticKeyword.Kind());
```

Comment:
>Assert.Equal(SyntaxKind.StaticKeyword, usingDirective.StaticKeyword.Kind()); [](http://example.com/codeflow?start=12&length=76)

It looks like there were changes around SemicolonToken. We should test that too.

Reply:
Good spot, thanks! Done.